locales/en-GB/messages.properties

```diff
@@ -86,16 +86,16 @@ fileSavingIndicator=Saving...
 fileSavingFailedIndicator=Saving failed, retrying...
 help=Help
 deleteProjectLink=Delete Project
-exportProjectLink=Export (.zip)
+exportProjectLink=Download files (.zip)
```

Comment:
I think we generate this file from pontoon, with only the en-US file being "Stable". Can you confirm this @gideonthomas?